addons/core/translations/resources/en-us.yaml

```diff
@@ -63,6 +63,7 @@ auth-method:
     finding: Finding
     always: Always
     primary: Primary
+    primary-method: Primary method?
```

Comment:
None blocking comment: Why change from Primary to Primary method? I do agree facilitates understanding for users, but I think it will have the side effect of doing the table cell wider? Not sure since I didn't try it but something to have in mind.

Reply:
Oh this particular change was design feedback to add a visible table header. Before the header was visually hidden.

Reply:
@lisbet-alvarez 

On my end it's fine, I just mentioned to have it in mind. If design is ok with it, let's do it!
I quickly check HDS documentation on table to "limit" a bit the width of the cell, but tbh I think we should leave the table to use its defaults.

Reply:
I think for this case the columns all adjusted to be wider to fill up the page.
In regards to the wording, of the columns it is based off of provided figma designs.